### PR TITLE
adds warning for 2fa users during org transfers

### LINKED ISSUE #22633: "Adding organizations to your enterprise" does not warn about problems with transferring non-2fa accounts
### Code of Conduct

- [X] I have read and agree to the GitHub Docs project's [Code of Conduct](https://github.com/github/docs/blob/main/CODE_OF_CONDUCT.md)

### What article on docs.github.com is affected?

https://docs.github.com/en/enterprise-cloud@latest/admin/user-management/managing-organizations-in-your-enterprise/adding-organizations-to-your-enterprise

### What part(s) of the article would you like to see updated?

The 'transferring an organization between enterprise accounts' section should have a warning to ensure customers are aware that non-2fa accounts being transferred into an enterprise that enforces 2fa policies will have issues (most likely getting removed from the org after transfer). 

### Additional information

_No response_

### PR DESCRIPTION
**Pull request by scarletspiderfan**
Adds a warning to let admins know that non-2FA accounts will be removed during an org transfer where the new enterprise enforces a 2FA requirement.

### Why:

Closes [ISSUE](https://github.com/github/docs/issues/22633)

### What's being changed (if available, include any code snippets, screenshots, or gifs):

 

I added a warning to the section for transferring organizations between enterprise accounts.

### Check off the following:

- [x] I have reviewed my changes in staging (look for the "Automatically generated comment" and click the links in the "Preview" column to view your latest changes).
- [x] For content changes, I have completed the [self-review checklist](https://github.com/github/docs/blob/main/contributing/self-review.md#self-review).

### CODE REVIEW

**Comment by Chiefy0x1** [comment]
> Adds a warning to let admins know that non-2FA accounts will be removed during an org transfer where the new enterprise enforces a 2FA requirement.
> 
> ### Why:
> Closes [ISSUE](https://github.com/github/docs/issues/22633)
> 
> ### What's being changed (if available, include any code snippets, screenshots, or gifs):
> I added a warning to the section for transferring organizations between enterprise accounts.
> 
> ### Check off the following:
> * [ ]  I have reviewed my changes in staging (look for the "Automatically generated comment" and click the links in the "Preview" column to view your latest changes).
> * [ ]  For content changes, I have completed the [self-review checklist](https://github.com/github/docs/blob/main/contributing/self-review.md#self-review).

Review required

**Comment by cmwilson21** [comment]
👋 @scarletspiderfan - Thanks for opening a PR and linking it to your issue ⚡ 

I'll get this triaged for review! :zap:

**Review by lecoursen — COMMENTED** [review]
Thanks for opening this PR @scarletspiderfan! This information would fit best in the "About addition of organizations to your enterprise account" section of this article, in the list of changes that apply after adding an existing organization your enterprise. Can you please move it there? Thanks!

**Comment by scarletspiderfan** [comment]
hi @lecoursen, I've removed the warning and added it to the top of the list instead. 👍

File: content/admin/user-management/managing-organizations-in-your-enterprise/adding-organizations-to-your-enterprise.md [hunk 1]
```diff
@@ -82,6 +83,7 @@ Enterprise owners can transfer existing organizations between enterprise account
 
 {% endnote %}
 
+
```
Comment (lecoursen) [inline content/admin/user-management/managing-organizations-in-your-enterprise/adding-organizations-to-your-enterprise.md:86]:
```suggestion
```

File: content/admin/user-management/managing-organizations-in-your-enterprise/adding-organizations-to-your-enterprise.md [hunk 2]
```diff
@@ -27,6 +27,7 @@ You can add new organizations to your enterprise account. If you do not use {% d
 
 After you add an existing organization to your enterprise, the organization's resources remain accessible to members at the same URLs, and the following changes will apply.
 
+- Organization users who do NOT use 2 factor authentication will be removed if 2FA is required by the Enterprise
```
Comment (lecoursen) [inline content/admin/user-management/managing-organizations-in-your-enterprise/adding-organizations-to-your-enterprise.md:30]:
```suggestion
- If two-factor authentication (2FA) is required by the enterprise, organization members who do not use 2FA will be removed from the organization.
```

**Review by lecoursen — DISMISSED** [review]
Thanks @scarletspiderfan! I'll apply some small suggestions and then merge. ⚡

**Review by lecoursen — APPROVED** [review]

**Pull request merged**